app/views/layouts/application.html.erb

```diff
@@ -6,14 +6,118 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbo-track': 'reload' %>
     <%= javascript_importmap_tags %>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Rubik+Mono+One&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b9c41a3878.js" crossorigin="anonymous"></script>
```

Comment:
I was wondering why font-awesome-sass wasn't in your Gemfile, and while this deviates from what the lesson teaches, I think this is fine because the usage of the `fa-*` classes is the same. Modern-day practices generally have apps deliver assets from their own pipeline instead of a CDN, so I'm curious why this way was chosen instead, but it's not that big a deal.